packages/headless/src/api/commerce/commerce-api-params.ts

```diff
@@ -74,3 +74,7 @@ export interface SelectedPageParam {
 export interface SelectedSortParam {
```

Comment:
Out of scope but since I've spotted it:

```suggestion
export interface SortParam {
```